src/components/FilterBar.jsx

```diff
@@ -1,42 +1,87 @@
 import { useEffect, useState } from "react";
 import { capitaliseString } from "../utils/utilityFunctions";
 import { useNavigate } from "react-router-dom";
-import { useParams } from "react-router-dom";
+import { useParams, useSearchParams } from "react-router-dom";
 import { getArticles } from "../utils/axiosFunctions";
 
 export default function FilterBar({ allTopics, setAllArticles, setLoading }) {
   const navigate = useNavigate();
+  const [searchParams, setSearchParams] = useSearchParams();
   const { topic = "all" } = useParams();
   const [currentTopic, setCurrentTopic] = useState(topic);
+  const [sortby = "created_at", setSortby] = useState(
+    searchParams.get("sortby")
+  );
```

Comment:
When you load the page, both of these are set to `null`, because this syntax is not quite working, you're trying to set the `sortby` to a default value in the destructured array, and initialising it to a value which evaluates to null (`searchParams.get("sortby")`)
Instead, you should initialise this state to "created_at"
The same applies to the order state